doc/news/changes/incompatibilities/20171117Bangerth

```diff
@@ -0,0 +1,15 @@
+Changed: The many KellyErrorEstimator::estimate() overloads all took
```

Comment:
all took -> all taken

Reply:
mmmm... I must say I agree with Wolfgang selection of `took` vs `taken`, i.e., `the overloads took as argument`.  `all taken` does not seem to be right w.r.t. what follows...

Reply:
Whoops. I think I meant to suggest `all take`

Reply:
But now reading further, I submit that even that is wrong because the function has had its arguments changed. Sorry, and please ignore!

Reply:
Actually, as far as I can tell all of the changes in the function signatures are, to date,
`typename FunctionMap<spacedim>::type` -> `typename FunctionMap<spacedim,typename InputVector::value_type>::type`. So `all take` is correct.